types/three/src/objects/InstancedMesh.d.ts

```diff
@@ -5,8 +5,11 @@ import { Mesh } from './Mesh';
 import { Matrix4 } from './../math/Matrix4';
 import { Color } from './../math/Color';
 
-export class InstancedMesh extends Mesh {
-    constructor(geometry: BufferGeometry, material: Material | Material[], count: number);
+export class InstancedMesh<
+    Geom extends BufferGeometry = BufferGeometry,
+    Mat extends Material | Material[] = Material
+> extends Mesh {
```

Comment:
We should add the generics each place we're extending from  `Mesh`.

```suggestion
> extends Mesh<TGeometry, TMaterial> {
```